### ISSUE TITLE
Badge request: flattr.com

### ISSUE THREAD

**Issue by Risto-Stevcev** [badges/shields#389]
url: https://flattr.com
example of the shield w/o shields.io: http://fabricjs.com/

**Comment by espadrine** [comment]
Thanks for suggesting!

API: http://developers.flattr.net/api/

**Comment by adjohnson916** [comment]
+1

**Comment by juliohm** [comment]
Hi, I see that commits were pushed, is this feature available already?

**Comment by paulmelnikow** [comment]
I'm afraid not. Looks like that PR was abandoned: #409.

Would you like to revive it? We have a good tutorial and a test framework, and if you have questions along the way you could post here.

**Comment by juliohm** [comment]
Thank you @paulmelnikow, I ended up with a simple workaround [here](https://github.com/juliohm/GeoStats.jl#contributing-and-supporting-). Not proud of it, but it works.

If I had better Javascript skills I would give it a shot and submit a PR.

**Comment by SethFalco** [comment]
For anyone that might look at this, note that there is no official public API anymore, and it's probably best to wait for whenever the next version comes around:

> We have closed down our API in order to prepare for future updates and exciting new features.
> 
> This means that the current Flattr API is no longer available and any applications using it (including our now defunct browser extension) no longer work.
> 
> But do not fear! We are not becoming yet another walled garden.
> We are already working on what the next version of our public API will look like.
> — https://flattr.com/api-notice

**Comment by calebcartwright** [comment]
> (Remember to consider the timestamp of this message if you're from the future.)

:laughing: well said

I'm not sure when that api notice was posted, but have updated the labels on this issue to reflect that it's blocked. I wish there was a timestamp on that notice and/or a ballpark for adding an API, but will leave this open for now. If down the road it is looking like there won't be an API then we can close as not actionable.

**Comment by SethFalco** [comment]
I messaged Flattr support about this, I'll paste a minimal excerpt of the ticket.
TL;DR: Don't expect this issue to be closed any time soon, unless as a `wontfix`.

> Currently we do not have any plans for a public API. …
> 
> — Support Staff (30th August 2021)

> … I’m sorry to say that this type of information will not be available any time soon. I’m not saying it won’t ever be available but it’s not a priority.
> 
> — Lead Developer (6th September 2021)

**Comment by PyvesB** [comment]
It's been many years and it sounds like this issue won't be actionable in any foreseeable future. I'll go ahead and close it, do feel free to reach out to us if ever the vendor situation changes. 😉
